en/reference/document-v1-api-reference.html

```diff
@@ -129,6 +129,8 @@ <h2 id="http-requests">HTTP requests</h2>
         <li><a href="#timeout">timeout</a></li>
         <li><a href="#tracelevel">tracelevel</a></li>
         <li><a href="#wanteddocumentcount">wantedDocumentCount</a></li>
+        <li><a href="#fromTimestamp">fromTimestamp</a></li>
+        <li><a href="#toTimestamp">toTimestamp</a></li>
```

Comment:
we lowercase the ID attributes normally
```suggestion
        <li><a href="#fromtimestamp">fromTimestamp</a></li>
        <li><a href="#totimestamp">toTimestamp</a></li>
```